.github/workflows/container-tests.yml

```diff
@@ -77,16 +77,39 @@ jobs:
         with:
           ref: "refs/pull/${{ github.event.issue.number }}/head"
 
+      - name: Prepare needed variables
+        shell: bash
+        id: vars
+        run: |
+          dockerfile="Dockerfile.${{ matrix.os_test }}"
+          if [[ ${{ matrix.os_test }} == "centos7" ]]; then
+            dockerfile="Dockerfile"
+          fi
+          echo "::set-output name=DOCKERFILE_NAME::${dockerfile}"
+
+      - name: Check ${{ matrix.version }}/${{ steps.vars.outputs.DOCKERFILE_NAME }} presents
+        id: check_dockerfile
+        uses: andstor/file-existence-action@v1
+        with:
+          files: "${{ matrix.version }}/${{ steps.vars.outputs.DOCKERFILE_NAME }}"
+
+      - name: Check .exclude-${{ matrix.os_test }} file presents
+        id: check_exclude_file
+        uses: andstor/file-existence-action@v1
+        with:
+          files: "${{ matrix.version }}/.exclude-${{ matrix.os_test }}"
+
       # https://github.com/sclorg/testing-farm-as-github-action
-      - name: Schedule tests on external Testing Farm by Testing-Farm-as-github-action
+      - name: Schedule tests for ${{ matrix.context }} - ${{ matrix.version }}
```

Comment:
What about switching version and context here (and in similar places)? So it is the same as in the name of the job?

Reply:
It makes sense here.